### ISSUE TITLE
Unblock request for saltwood.top

### ISSUE THREAD

**Issue by SALTWOOD** [cbuijs/hagezi#4]
I’m the owner of the domain saltwood.top, and I recently noticed it’s been added to an ad block blacklist. I believe this happened because I ran a BT tracker on it in the past. At the time, I didn’t realize the potential issues this could cause.

As soon as I became aware of the problem, I removed the BT tracker completely from my domain. I’d like to confirm that it’s no longer being used for anything related to that.

Could you please review my domain and consider removing it from the blacklist? Let me know if you need any more details from me.

Thanks a lot for your help! Let me know immediately if you need any more information or have concerns.

I’m not fluent in English and used AI to assist with this issue. Please excuse any errors in grammar or wording. Thanks!

**Comment by cbuijs** [comment]
Hi @SALTWOOD,

I am just repacking/optimizing the Hagezi lists here, I do not maintain the content/list itself.

You can mention/report it on the below link, the guy running it (Gerd), is very responsive:
https://github.com/hagezi/dns-blocklists/issues

Cheers,
-Chris.

**Comment by SALTWOOD** [comment]
Thanks. I opened a new issue there, and thanks for your reply.
